cs/http-request-response.texy

```diff
@@ -13,7 +13,48 @@ Nette Framework pročistí data odeslané uživatelem od kontrolních a neplatn
 HTTP požadavek, což je objekt třídy [Nette\Http\Request |api:], nevytváříme přímo, ale získáme jako službu z [DI kontejneru |di-usage].
 
 /--php
-$httpRequest = $container->getByType('Nette\Http\Request');
+<?php
+
+use Nette\Application\UI\Presenter;
+use Nette\Http\Request;
+
+class MySuperService
+{
+
+	/**
+	 * @var Request
+	 */
+	private $httpRequest;
+
+
+	public function __construct(Request $httpRequest)
+	{
+		$this->httpRequest = $httpRequest;
+	}
+
+}
+\--
+
+Presenter má HTTP request k dispozici v předkovi. Vyžádat si ho můžeme metodou `Nette\Application\UI\Presneter::getHttpRequest()`:
+
+/--php
+<?php
```

Comment:
I think it's not needed to show presenter with `getHttpRequest`. It's mention right above. WDYT @paveljanda ?